functions/tree_calculation.py

```diff
@@ -0,0 +1,41 @@
+import colorama
+from colorama import Fore
+colorama.init()

```

Comment:
These are libraries print colored terminal text, can move with the print lines to the script later too if needed